spieleplattform-frontend/src/pages/Game/GamePage.tsx

```diff
@@ -1,27 +1,38 @@
 import {useParams} from "react-router-dom";
 import Game from "./Game";
-import Rating from "../Rating/Rating";
+import {useEffect, useState} from "react";
+import axios from "axios";
 
 export default function GamePage() {
     const {id} = useParams()
-    
-    const game: Game = {
-        name: "Minecraft",
-        releaseDate: new Date(2011, 11, 18),
-        developer: "Mojang Studios",
-        ratings: [
-            {
-                ratingValue: 5,
-                comment: "Best game ever!",
-            }
-        ]
-    }
-    
+
+    const [game, setGame] = useState<Game | null>(null);
+    const [error, setError] = useState(null);
+
+    useEffect(() => {
+        axios.get("/game/" + id)
+            .then((response) => {
+                setGame(response.data)
+            })
+            .catch(error => {
+                setError(error);
+            })
+    }, [id])
+
+    if (error) return <h1>Game not found</h1>
```

Comment:
Dein return hier, hattest du doch in eine eigene Komponente ausgelagert, die du hier verwenden könntest. Zusätzlich, könntest du hier noch eine Condition einbauen, die den Statuscode prüft und nur not found anzeigt, wenn du auch wirklich eine 404 zurückbekommst.